llvm/test/CodeGen/AArch64/get-active-lane-mask-extract.ll

```diff
@@ -1,4 +1,4 @@
-; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --filter-out "kill:" --version 4
```

Comment:
I don't think this is a wise move as it prevents the use of `-NEXT` markers, meaning we can miss new instructions being emitted. I think kill flags are just a fact of life for these tests.

Reply:
Fair enough, fixed.